docs/diginstroom/sip/1.0/2_terminology.md

```diff
@@ -57,16 +57,17 @@ In the context of noting metadata values, the following datatypes are mentioned
 
 | Datatype       | Definition |
```

Comment:
Shouldn't these be ordered alphabetically?

Reply:
I grouped types related to dates and types related to identification, but alphabetically is fine as well. Up to you!